terraform-project-api-gateway_module/main.tf

```diff
@@ -79,11 +72,19 @@ data "aws_ssm_parameter" "api_gateway_cs_lambda_authorizer_cognito_user_pool_id"
   name = "arn:aws:ssm:${data.aws_ssm_parameter.shared_service_region.value}:${data.aws_ssm_parameter.shared_service_account_id.value}:parameter/unity/cs/routing/venue-api-gateway/cs-lambda-authorizer-cognito-user-pool-id"
 }
 
-# Unity CS Common Lambda Authorizer Allowed Cognito User Groups List (Command Seperated)
+# Unity CS Common Lambda Authorizer Allowed Cognito User Groups List (Comma Seperated)
 data "aws_ssm_parameter" "api_gateway_cs_lambda_authorizer_cognito_user_groups_list" {
   name = "arn:aws:ssm:${data.aws_ssm_parameter.shared_service_region.value}:${data.aws_ssm_parameter.shared_service_account_id.value}:parameter/unity/cs/routing/venue-api-gateway/cs-lambda-authorizer-cognito-user-groups-list"
 }
 
+# Unity Management Console NLB
+data "aws_lb" "unity_mc_nlb" {
```

Comment:
Could this result in finding multiple, or the wrong NLB, if multiple have these tags?

Reply:
@galenatjpl I updated the code to search using load balance name.